content/pull-requests/collaborating-with-pull-requests/working-with-forks/allowing-changes-to-a-pull-request-branch-created-from-a-fork.md

```diff
@@ -38,6 +38,8 @@ You can set commit permissions when you first create a pull request from a fork.
 1. In the list of pull requests, navigate to the pull request that you'd like to allow commits on.
 {% data reusables.repositories.allow-maintainers-user-forks %}
 
+![Screenshot of a Pull request. On the right bottom side, a blue, clicked checkbox, labelled "Allow edits and access to secrets by maintainers".](/assets/images/help/pull_requests/checkbox-for-pull-request-edit-access.png)
```

Comment:
Our images or screenshots must be taken with the GitHub default light theme on (for more details, see https://github.com/settings/appearance. You'll need to select "Light default" before taking the screenshot), and also the orange outline has a specific line thickness and a specific round corner measurement.
We may need to provide you with the screenshot ourselves. Let me investigate.